src/receipt-capture-plugin.ts

```diff
@@ -6,18 +6,18 @@
 import type { Account } from './account';
 import type { Receipt } from './receipt';
 
-
 export type ScanType = 'PHYSICAL' | 'EMAIL' | 'RETAILER' | 'ONLINE';
 
 export interface ReqAccount {
-  username: string,
-  password: string,
-  source: string,
+  username: string;
+  password: string;
+  source: string;
 }
 
-export interface ReqInitialize{
+export interface ReqInitialize {
   licenseKey: string;
   productKey: string;
+  googleId: string | undefined;
```

Comment:
@JesseMonteiro & @ricardobrg can you explain what this is and why it's in the Initialization obj?

Reply:
Looks like it is missing the typescript documentation and readme. I need to add those.

This is the Google Client Id for Google OAuth. If it is not provided IMAP will be used for Gmail.

Reply:
see [4b86e42](https://github.com/tiki/capture-receipt-capacitor/pull/265/commits/4b86e4281595c672aa4267e3b8e0fcf120872ffc)